index.html

```diff
@@ -75,7 +75,14 @@
     company: "Digital Bazaar",
     companyURL: "https://digitalbazaar.com/",
     w3cid: 41758
-  }]
+  }, {
+    name: "Jonathan Holt, DO, MS", 
+    url: "https://www.linkedin.com/in/jonathan-holt-do-ms/", 
+    company: "TranSendX", 
+    companyURL: "https://transendx.com", 
+    w3cid: 115670 
+  }
+  ]
```

Comment:
The Chairs have requested that PR authors do not modify the Editor/Author lists until later in the process. Please remove this change set.

/cc @brentzundel @burnburn

Reply:
Removed, however given the substantive contribution regarding the CDDL and the work that I have put in, seems only fair to consider me an author.